app/_src/policies/meshloadbalancingstrategy.md

```diff
@@ -237,6 +269,239 @@ Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](/doc
+
+{% endtab %}
+{% endtabs %}
+
+### Route within the local zone equally, but specify cross zone order
+
+Requests to the backend service will be evenly distributed among all endpoints within the local zone. If there are fewer than 25% healthy hosts in the local zone, traffic will be redirected to other zones. Initially, traffic will be sent to the `us-1` zone. In the event that the `us-1` zone becomes unavailable, traffic will then be directed to all zones, except for `us-2` and `us-3`. If these zones are also found to have unhealthy hosts, the traffic will be rerouted to `us-2` and `us-3`.
+
+{% tabs cross-zone-backend useUrlFragment=false %}
+{% tab cross-zone-backend Kubernetes %}
+
+```yaml
+kind: MeshLoadBalancingStrategy
+apiVersion: kuma.io/v1alpha1
+metadata:
+  name: cross-zone-backend
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: mesh-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        localityAwareness:
+          crossZone:
+            failover:
+              - to:
+                  type: Only
+                  zones: ["us-1"]
+              - to:
+                  type: AnyExcept
+                  zones: ["us-2", "us-3"]
+              - to:
+                  type: Any
+          failoverThreshold:
+            percentage: 25
+```
+
+Apply the configuration with `kubectl apply -f [..]`.
+
+{% endtab %}
+{% tab cross-zone-backend Universal %}
+
+```yaml
+type: MeshLoadBalancingStrategy
+name: cross-zone-backend
+mesh: mesh-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        localityAwareness:
+          crossZone:
+            failover:
+              - to:
+                  type: Only
+                  zones: ["us-1"]
+              - to:
+                  type: AnyExcept
+                  zones: ["us-2", "us-3"]
+              - to:
+                  type: Any
+            failoverThreshold:
+              percentage: 25
+```
+
+Apply the configuration with `kumactl apply -f [..]` or with the [HTTP API](/docs/{{ page.version }}/reference/http-api).
+
+{% endtab %}
+{% endtabs %}
+
+### Prioritize traffic to dataplanes within the same datacenter and fallback cross zone in specific order
+
+Requests to backend will be distributed based on weights, with 99.9% of requests routed to data planes in the same datacenter, 0.001% to data planes in the same region, and the remainder to other local instances.
+
+When there are no healthy backends within the local zone, traffic from data planes in zones `us-1`, `us-2`, and `us-3` will only fall back to zones `us-1`, `us-2`, and `us-3`. If there are no healthy instances in these zones, requests from any instance will then fall back to `us-4`. If there are no healthy instances in `us-4`, the request will fail, as the last rule, by default, has type: `None`, meaning no fallback is allowed.
```

Comment:
The "last rule" having type: None makes sense, but I think it's not mentioned anywhere else. Should we add it somewhere before giving examples?

Why isn't `eu` mentioned in this explanation even though it's defined [here](https://github.com/kumahq/kuma-website/pull/1506/files#diff-a9f8e0fd2b5222ab80b0bf763aefff4b9387802a05f2ad757de7a036815beb18R448)?

nitpick: 
```suggestion
When no healthy backends are available within the local zone, traffic from data planes in zones `us-1`, `us-2`, and `us-3` will only fall back to zones `us-1`, `us-2`, and `us-3`. If there are no healthy instances in these zones, requests from any instance will then fall back to us-4. If there are no healthy instances in us-4, the request will fail, as the last rule, by default, has a type of None, meaning no fallback is allowed.
```